src/presenter/films-list-presenter.js

```diff
@@ -25,24 +33,50 @@ export default class FilmsListPresenter {
 
   prepearFilms = (films) => films;
 
+
+  #renderedFilmCount = TASK_COUNT_PER_STEP;
+
+  #loadMoreButtonComponent = new LoadMoreButtonView();
+
   init = (filmsModel, commentsModel) => {
-    this.filmsModel = filmsModel;
-    this.commentsModel = commentsModel;
-    this.cards = this.prepearFilms([...this.filmsModel.getFilms()]);
+    this.#filmsModel = filmsModel;
+    this.#commentsModel = commentsModel;
+    this.#cards = this.prepearFilms([...this.#filmsModel.get()]);
+
+    render(this.#filmsListView, this.#container.element);
 
-    render(this.filmsListView, this.container.getElement());
     render(
       this.filmsListContainerView,
-      this.filmsListView.getElement()
+      this.#filmsListView.element
     );
 
-    this.cards.forEach((film) => {
-      this.createFilm(film);
+    this.#cards.slice(0, 5).forEach((film) => {
```

Comment:
Либо константа TASK_COUNT_PER_STEP, либо общее количество фильмов, в зависимости от того как ты будешь перерисовывать страницу в дальнейшем, сейчас не важно 
```suggestion
    this.#cards.slice(0, this.#renderedFilmCount).forEach((film) => {
```